legal-api/src/legal_api/services/document_meta.py

```diff
@@ -388,8 +388,7 @@ def create_report_object(self, title: str, filename: str, report_type=None):
 
     def get_general_filename(self, name: str):
         """Return a general filename string."""
-        filing_date_str = LegislationDatetime.format_as_legislation_date(self._filing_date)
-        file_name = f'{self._business_identifier} - {name} - {filing_date_str}.pdf'
+        file_name = f'{self._business_identifier} - {name} - {self._filing_date}.pdf'
```

Comment:
document_meta.py is used in v1 (which is obsolete). Not fixing the legislation date